.gitignore

```diff
@@ -2,3 +2,4 @@ pkg
 .DS_Store
 *~
 rdoc/
+*.gem
```

Comment:
I would like to keep copies of all published gem files in this repo as a backup. Whatever happens to rubygems this repo could be used to verify the integrity of data hosted on rubygems and restore it if needed.

Reply:
changed this in #3

Reply:
Understood, thank you.